packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -278,6 +278,29 @@ class CreateCommand extends FlutterCommand {
     return template;
   }
 
+  Set<Uri> get templateManifest => _templateManifest ??= _computeTemplateManifest();
+  Set<Uri> _templateManifest;
+  Set<Uri> _computeTemplateManifest() {
+    final String flutterToolsAbsolutePath = globals.fs.path.join(
+      Cache.flutterRoot,
+      'packages',
+      'flutter_tools',
+    );
+    final String manifestPath = globals.fs.path.join(
+      flutterToolsAbsolutePath,
+      'templates',
+      'template_manifest.json',
+    );
+    final Map<String, Object> manifest = json.decode(
+      globals.fs.file(manifestPath).readAsStringSync(),
+    ) as Map<String, Object>;
+    return Set<Uri>.from(
+      (manifest['files'] as List<Object>)
+        .cast<String>()
+        .map<Uri>((String path) => Uri.file(globals.fs.path.join(flutterToolsAbsolutePath, path))),
```

Comment:
if that string is not a valid path, `Uri.file()` will throw. I'm thinking we should catch that, especially since we're re-constructing the absolute path.

Reply:
oh wait, nvm. i see this is checked in.